[Delivers #171293079] sala de estar mapa

JuanmaSalva opened the pull request:
Ya se leen todos los puntos de spawn por tiled y tmb he cambiado la textura del router sry (esto obviamente es provisional)

JuanmaSalva committed:
Primer commit sala de estar

JuanmaSalva committed:
Tenemos fondo

JuanmaSalva committed:
Casi esta (en teoria)

JuanmaSalva committed:
Falta nada

Tenemos un problema con la escala de las paredes de la derechay la de arriba ESPAÑA

Co-Authored-By: will7461 <will7461@users.noreply.github.com>
Co-Authored-By: adrian-de-lucas-gomez <adrian-de-lucas-gomez@users.noreply.github.com>

AdriAlvarez00 committed:
Arreglado el cambio de posición de tiled a box2d

JuanmaSalva committed:
Spawns via Tiled

Ya hay una capa en tiled que se llama spawns q todos los abjetos que hay alli son, obvio, spawns

JuanmaSalva committed:
Spawns arreglados y vida

La vida ya se ve por encima de las paredes

JuanmaSalva committed:
[Finishes #171293079] descripcion

Cuando se crea el tilemap se leen todas las capas de objetos y se guardan en sus vectores correspondientes que luego son accesibles desde sus métodos correspondientes que estan en el tilemap. De momento va con id`s pero se podria cambiar para que sea aleatorio

JuanmaSalva committed:
Merge branch 'master' into 171293079-Sala-de-estar-MAPA

JuanmaSalva committed:
Merge branch 'master' into 171293079-Sala-de-estar-MAPA

JuanmaSalva committed:
El router ya sale segun tiled

tambien he cambiado la textra del router q me pone nervioso la textura por defecto

EstebenR reviewed (changes requested):
no me puedo agarrar a las paredes :c

WillMDev committed:
no agarre bug pendiente

WillMDev committed:
Merge branch 'master' into 171293079-Sala-de-estar-MAPA

WillMDev committed:
Arreglado bug de agarre y json de SalaDeEstar

JuanmaSalva committed:
Merge branch '172062249-InputFix' into 171293079-Sala-de-estar-MAPA

JuanmaSalva committed:
Terminado

El fondo ya es una textura y la manera de cargar las cosas del json esta mas bonita

EstebenR committed:
Merge branch 'master' into 171293079-Sala-de-estar-MAPA

EstebenR reviewed (approved):
Molto bene props al equipo

eljorgemo reviewed (approved):
Muy chido
